.github/workflows/keepalive.yml

```diff
@@ -0,0 +1,42 @@
+name: Keep scheduled workflows alive
+
+on:
+  schedule:
+    - cron: "16 07 10 * *"
+  workflow_dispatch:
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: 'true'
+        env:
```

Comment:
This `env` block probably needs to be moved inside the `Create and push a commit` step (the `actions/checkout` step cannot make use of the `user.name`/`user.email` configuration).